<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConveyorControl</class>
 <widget class="QDialog" name="ConveyorControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1396</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_off">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: none;
    border: none;
	padding-left: 10px;
    background-color: rgba(0, 0, 0, 0); 
}
</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/stop_resized.jpg</normaloff>:/images/stop_resized.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="cameraLabel_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>240</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_return">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: none;
    border: none;
    background-color: rgba(0, 0, 0, 0); /* fully transparent */
}
</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/exit_resized-removebg-preview.png</normaloff>:/images/exit_resized-removebg-preview.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_on">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: none;
    border: none;
    background-color: rgba(0, 0, 0, 0); 
	font-size: 22pt;
	border: 2px solid rgba(0, 0, 0, 0.4); 
    border-radius: 12px; 
}
</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="cameraLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>240</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
